### PR TITLE
Feature/calculator components

### PR DESCRIPTION
**Pull request by Braingenius2**
# Added calculator components (UI only)
In this pull request, I implemented the following:

- [x] deleted all the boilerplate from Create React App (text, images, styles)
- [x] created a new directory called components.
- [x] created a new Calculator.js file inside components.
- [x] implemented the design only (JSX and CSS). The math logic to make it work will be added in the next project.
- [x] used functional components

### CODE REVIEW

File: src/css/Calculator.css [hunk 1]
```diff
@@ -0,0 +1,63 @@
+.calculator {
+  display: flex;
+  flex-direction: column;
+  width: 40%;
+  background-color: #333;
+  border-radius: 5px;
+  padding: 10px;
+}
+
```
Comment (binhussen) [inline src/css/Calculator.css]: - Great Job for having this beautiful UI, I think for this milestone you have to stick to the given design to fulfill the [requirement](https://github.com/microverseinc/curriculum-react-redux/blob/main/math-magicians/project_components.md) 👍 
#### Your design 👇
Screenshot 2023-07-05 152409
#### Given Design 👇
Screenshot 2023-07-05 152504

**Review by binhussen — CHANGES_REQUESTED** [review]
Hi @Braingenius2 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights

- Arranged Component ✔️
- Well Structured Code✔️
- Used Git flow ✔️

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by ShadyShawkat — APPROVED** [review]
## Hi @Braingenius2, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## To Highlight
- Correct use of GitFlow ✔️ 
- Descriptive commit messages ✔️ 
- Professional looking pull request ✔️ 
- Your readme looks professional ✔️ 

Image

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Alusp — COMMENTED** [review]
Hi @Braingenius2, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
congrats

✅ Implemented Calculator Component
✅ Linters are passing
✅ Well descriptive PR

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @Alusp in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**